src/components/TimerControls.styled.tsx

```diff
@@ -0,0 +1,12 @@
+import styled from "styled-components";
```

Comment:
Could you move the `TimerControls.styled.tsx` and `TimerControls.tsx` files to a folder?